ocs_ci/deployment/helpers/lso_helpers.py

```diff
@@ -471,6 +471,7 @@ def cleanup_nodes_for_lso_install():
 
     nodes = get_all_nodes()
     node_objs = get_node_objs(nodes)
+    ocp.OCP().exec_oc_cmd("project default")
```

Comment:
can we use https://github.com/red-hat-storage/ocs-ci/blob/fe13f663a07e918744083712b1d6904a09660c89/ocs_ci/ocs/ocp.py#L1319

Reply:
Also do we need to switch back to original project ( openshift-storage ) ?

Reply:
based on logs the original ns is `baremetal-debug-ns`